### ISSUE TITLE
1-Rules Improvement

### ISSUE THREAD

**Issue by josiahls** [unccv/learning_to_see#1]
So I created a new rule:

```python
rules.append(np.array(([[0, 0, 0, 1, 1, 1, 1, 1, 0],
                        [0, 0, 0, 1, 1, 1, 1, 1, 0],
                        [0, 0, 0, 1, 1, 1, 1, 1, 0],
                        [0, 0, 0, 1, 1, 1, 1, 0, 0],
                        [0, 0, 1, 1, 1, 1, 1, 0, 0],
                        [0, 0, 1, 1, 1, 1, 0, 0, 0],
                        [0, 1, 1, 1, 1, 1, 0, 0, 0],
                        [0, 1, 1, 1, 1, 1, 0, 0, 0],
                        [0, 1, 1, 1, 1, 1, 0, 0, 0]])))
```
And I currently don't see an improvement in recognizing fingers being tilted to the right. (Note, I am trying other versions of this, just trying to find fingers that are tilted)
I know that you have a way to snapshoting parts of the image with:
```python
exampleToShow = X[600,:].reshape(9,9)
``` 

However, doesn't this take a horizontal line and compress it into a 9x9 matrix? Because this looks like the right side of a finger, but I'm not sure if that is actually what that represents?

Basically:
- Is there is a faster way to look at the small sections of the image to copy into an array?
- Are we supposed to toggle the rules array directly? (which is slow) or is there an easy way to snapshot parts of a hand image to use as rule? 

I'm thinking adding a util function:
```python

bool_slice_of_image = get_slice(image=image, anchor=[5,5], size=[9, 9])
```
that then outputs:
```python
[[0 0 0 0 0 1 1 1 1]
 [0 0 0 0 0 1 1 1 1]
 [0 0 0 0 0 1 1 1 1]
 [0 0 0 0 1 1 1 1 1]
 [0 0 0 0 1 1 1 1 1]
 [0 0 0 0 1 1 1 1 1]
 [0 0 0 0 1 1 1 1 1]
 [0 0 0 0 1 1 1 1 1]
 [0 0 0 0 1 1 1 1 1]]
```
